src/components/pill-selector-dropdown/PillSelector.scss

```diff
@@ -104,6 +103,8 @@
             min-width: 0;
             overflow: hidden;
             padding: 5px 3px 3px;
+            resize: none;
+            white-space: nowrap;
```

Comment:
This will prevent the input from wrapping to multiple lines so the behavior will be identical to the text input.